src/builders.ts

```diff
@@ -88,11 +90,9 @@ function getRuleId(message: any) {
 }
 
 function getCreatedDate(): Date {
-  if (process.env.TODO_CREATED_DATE) {
-    return new Date(process.env.TODO_CREATED_DATE);
-  }
+  const date = process.env.TODO_CREATED_DATE ? new Date(process.env.TODO_CREATED_DATE) : new Date();
```

Comment:
This is mostly just for testing right?

Reply:
Ya this is precisely so we can pass dates in via `execa` to the child process' env.